smart-connector-api/src/main/java/eu/knowledge/engine/smartconnector/api/AskResult.java

```diff
@@ -99,8 +103,12 @@ public ReasonerPlan getReasonerPlan() {
 		return this.reasonerPlan;
 	}
 
+	public KnowledgeGapSet getKnowledgeGaps() {
+		return this.knowledgeGaps;
+	}
+
 	@Override
 	public String toString() {
-		return "AskResult [bindings=" + bindings + ", exchangeInfoPerKnowledgeBase=" + exchangeInfos + "]";
+		return "AskResult [bindings=" + bindings + ", exchangeInfoPerKnowledgeBase=" + exchangeInfos + ", knowledgeGaps=" + knowledgeGaps + "]";
```

Comment:
This addition currently does not do much, since the `KnowledgeGapSet` has no custom `toString()` method, so it just shows something like `KnowledgeGapSet@23423423`. But, if we decide to remove this class altogether (see comment below) then this is of course not necessary 👍

Reply:
Well, it does add information in the AskResult class which is being logged and shown as INFO at the `ProactiveApiServiceImpl` class upon receipt of the AskResult.

Reply:
Ok, maybe I didn't look correctly, but I expected it to just show something like `KnowledgeGapSet@23423423` after the `knowledgeGaps=`. But if the `KnowledgeGapSet` is replaced by `Set<KnowledgeGap>` the `toString(...)` is more informative.